index.js

```diff
@@ -36,12 +50,18 @@ window.onload = () => {
 
     if(location.host === urls.prod) {
         !firebase.apps.length ? firebase.initializeApp(prodFirebaseConfig()) : firebase.app();
+        window.DD_RUM && window.DD_RUM.init({ ...DatadogDevConfig, env: 'prod' });
+        window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
     }
     else if(location.host === urls.stage) {
         !firebase.apps.length ? firebase.initializeApp(stageFirebaseConfig()) : firebase.app();
+        window.DD_RUM && window.DD_RUM.init({ ...DatadogDevConfig, env: 'stage' });
+        window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
     }
     else {
         !firebase.apps.length ? firebase.initializeApp(devFirebaseConfig()) : firebase.app();
+        window.DD_RUM && window.DD_RUM.init(DatadogDevConfig);
```

Comment:
Just a suggestion from personal preference... Declare the Datadog variable without an "env" parameter and don't name it "dev config", then in code always pass which environment you will be using. Might help clarify that the config is truly for all tiers.

Reply:
Updated.